sorted_rating.sh

```diff
@@ -0,0 +1,30 @@
+echo enter directory which contains movie folder
+read dir
```

Comment:
Try using command line arguments here instead of asking for input using prompt. Read this: https://grofers.slack.com/archives/bootcamp/p1465012408000095